src/site/content/en/design-system/colors.njk

```diff
@@ -5,11 +5,48 @@ permalink: '/design-system/colors/index.html'
 {% extends "design-system.njk" %}
 {% set title = 'Colors' %}
 {% set summary %}
-The colors and how to use them Nullam quis risus eget urna mollis ornare vel eu leo. Donec id elit non mi porta gravida at eget metus.
+Colors are set with design tokens, which are located here: `src/site/_data/design/tokens.json`. 
+There is also a contextual relationship between colors and how they are applied with themes: `src/site/_data/design/themes.js`
+
+### Todo
+
+- Create proper color references 
+- Document themeing 
+- Document color usage 
+
 {% endset %}
 {% set breadcrumbs = [
   {
     title: 'Colors',
     url: page.url
   }
 ]%}
+{% block inner %}
+{% for group in design.tokens.colors %}
+  <h2>{{ group.group }}</h2>
+  <!-- TEMP LAYOUT -->
+  <div class="swatches">
+    {% for color in group.items %}
+      <div>
+        <h3>{{ color.name }}</h3>
+        <dl>
```

Comment:
OK. Feel free to say no, but...

I was wondering if we can make the swatches align 😅

![image](https://user-images.githubusercontent.com/1066253/124964354-c6b96900-dfd5-11eb-9a40-ab7e606c5e04.png)

I think the only way to do it would be to make the dl's wrap
![image](https://user-images.githubusercontent.com/1066253/124964781-42b3b100-dfd6-11eb-8297-776dfbb1c3b9.png)

or make the `main` element wider.

I know it's not important...but it looks so tidy when they all line up...

Reply:
Oh lol don't worry about this mate! I've literally just dumped stuff on a page so I could copy and paste into my theme config 😂

I'll be designing a proper UI for all of these sections of the design system once I've got some fundamental CSS done!

Reply:
ok cool!